src/cli_plugin/install/cleanup.js

```diff
@@ -0,0 +1,32 @@
+import rimraf from 'rimraf';
+import fs from 'fs';
+
+export function cleanPrevious(settings, logger) {
+  return new Promise(function (resolve, reject) {
+    try {
+      fs.statSync(settings.workingPath);
+
+      logger.log('Found previous install attempt. Deleting...');
+      try {
+        rimraf.sync(settings.workingPath);
+      } catch (e) {
+        reject(e);
+      }
+      resolve();
+    } catch (e) {
+      if (e.code !== 'ENOENT') reject(e);
+
+      resolve();
+    }
+  });
+};
+
+export function cleanError(settings) {
```

Comment:
What does this implementation have to do with errors? Perhaps another name would be more appropriate?